### ISSUE TITLE
Consider adding test data to the PyPi distribution

### ISSUE THREAD

**Issue by toolness** [aepyornis/nyc-db#55]
Hello again!

I really like the [integration test data](https://github.com/aepyornis/nyc-db/tree/master/src/tests/integration/data) for this project.  It's been very useful for building tests for any project that uses NYCDB, such as [Who Owns What](https://github.com/JustFixNYC/who-owns-what), as well as some other experiments of mine.

However, the test data isn't in the PyPI distribution of the project (at least, I don't think it is).  This means that for all my projects I've had to download the ZIP file version of the repository to get the data.

Since the data is only about a megabyte or two in size, I'm curious if it'd be helpful to developers to have it just ship with the PyPI distribution, along with documentation so that devs know it's there?  No worries if not, I could be the only person who finds it handy.

**Comment by aepyornis** [comment]
yeah, I think the test and test data are excluded from the distribution. I don't see any problem with including them!
